paddle/fluid/framework/program_desc.h

```diff
@@ -55,6 +56,9 @@ class ProgramDesc {
   const std::vector<std::string> GetFeedTargetNames();
   const std::vector<std::string> GetFetchTargetNames();
 
+  void SetFeedHolderName(const std::string &feed_holder_name);
```

Comment:
Is feed/fetch holder a new concept? add some comments will be better.

Reply:
Done.